tiatoolbox/utils/misc.py

```diff
@@ -202,36 +202,60 @@ def get_luminosity_tissue_mask(img, threshold):
     return tissue_mask
 
 
+@np.vectorize
+def mpp2common_objective_power(
+    mpp, common_powers=(1, 1.25, 2, 2.5, 4, 5, 10, 20, 40, 60, 90, 100)
+):
+    """Approximate (commonly used round value) of objective power from mpp.
+
+    Args:
+        mpp (float or tuple of float): Microns per-pixel.
+        common_powers (list of float): A sequence of objective
+            power values to round to.
+
+    Returns:
+        float: Objective power approximation.
+
+    Raises:
+        ValueError
+    """
+    op = mpp2objective_power(mpp)
+    distances = [np.abs(op - power) for power in common_powers]
+    closest_match = common_powers[np.argmin(distances)]
+    return closest_match
+
+
+@np.vectorize
+def objective_power2mpp(objective_power):
+    """Approximate mpp from objective_power.
+
+
+    Args:
+        objective_power (float or tuple of float): Objective power.
+
+    Returns:
+        float: Microns per-pixel (MPP) approximation.
+
+    Raises:
+        ValueError
+    """
+    return 10 * (np.float(objective_power) ** -1)
+
+
+@np.vectorize
 def mpp2objective_power(mpp):
```

Comment:
I cannot understand this function. This function says mpp2objective but it's returning objectivepower2mpp?

Reply:
It is a self-inverse function hence the inverse calls the other.